core/tutorials/buggyamb/Docs/load_generator.md

```diff
@@ -62,6 +62,6 @@ For example I made two requests to the third slow scenario and four requests to
 
 ![BuggyAmb Load Generator](Images/load_generator_crash_symptoms.png)
 
-I got HTTP 503 - Sevice Unavailable error for one of my requests and there are other requests ended up with ```error```. I am hosting BuggyAmb on IIS and I may get HTTP 503 if the application pool is disabled or some limits are achieved (e.g.: queue is full). Queue should not be full because there are not too many requests so it should be a crash symptom and I can confirm that in event logs. 
+I got HTTP 503 - Sevice Unavailable error for one of my requests and there are other requests ended up with ```error```. I am hosting BuggyAmb on IIS and I may get HTTP 503 if the application pool is disabled or some limits are achieved (e.g.: queue is full). Queue should not be full because there are not too many requests so it should be a crash symptom and I can confirm that in event logs.
```

Comment:
```suggestion
I got HTTP 503 - Sevice Unavailable error for one of my requests, and there are other requests that ended up with `error`. I am hosting BuggyAmb on IIS and I may get HTTP 503 if the application pool is disabled or some limits are achieved (for example, the queue is full). Queue should not be full because there are not too many requests, so it should be a crash symptom. I can confirm that in event logs.
```